src/lib/utils/options/declaration.ts

```diff
@@ -293,3 +315,59 @@ function getMapError(map: MapDeclarationOption<unknown>['map'], name: string) {
 
     return `${name} must be one of ${keys.join(', ')}`;
 }
+
+/**
+ * Returns an error message for a value that is out ot bounds of the given min and/or max values.
+ * @param name The name of the thing the value represents.
+ * @param minValue The lower bound of the range of allowed values.
+ * @param maxValue The upper bound of the range of allowed values.
+ * @returns The error message.
+ */
+function getBoundsError(name: string, minValue?: number, maxValue?: number): string {
+    if (isFiniteNumber(minValue) && isFiniteNumber(maxValue)) {
+        return `${name} must be between ${minValue} and ${maxValue}`;
+    } else if (isFiniteNumber(minValue)) {
+        return `${name} must be >= ${minValue}`;
+    } else if (isFiniteNumber(maxValue)) {
+        return `${name} must be <= ${maxValue}`;
+    } else {
+        return '';
+    }
+}
+
+/**
+ * Checks if the given number option has bounds specified.
+ * @param numberOption The number option being checked.
+ * @retursn True, if the given number option has bounds specified, otherwise false.
+ */
+function hasBounds(numberOption: NumberDeclarationOption): boolean {
+    return isFiniteNumber(numberOption.minValue) || isFiniteNumber(numberOption.maxValue);
+}
+
+/**
+ * Checks if the given value is a finite number.
+ * @param value The value being checked.
+ * @returns True, if the value is a finite number, otherwise false.
+ */
+function isFiniteNumber(value?: unknown): value is number {
+    return typeof value === 'number' && isFinite(value);
```

Comment:
Well this sent me on a wild goose chase. This is equivalent to `Number.isFinite`, but the types for that function say they only accept `number`, so TS would yell... this was fixed, but the fix changed generated files instead of the source files and so was thrown out in the next release.